Technical-Steering-Committee/projects.md

```diff
@@ -0,0 +1,180 @@
+
+- Under Development
+- Technical Preview
+- Generally Available (GA)
+- Long Term Support (LTS)
+- Deprecated
+- Archived
+
+![Life Cycle Transitions](projects-lifecycle.png "Life Cycle Transitions")
+
+### Under Development
+
+#### Purpose
+
+To let developers know that something is going on and that they can join the effort. To let the users and extenders know that such a project may become available in the future. 
+
+#### Timeline
+
+As long as it takes to move the project to at least the Technical Preview stage.
+
+#### Support
+
+The project in this stage isn't ready to be deployed, and as such, there is no support other than reaching out directly to the developers. 
+
+#### Recommended Use
+
+Help with the development of the functionality. 
+
+#### Possible transitions
+
+- From Under Development to Technical Preview means that the project is ready for validation by customers
+- From Under Development to Generally Available means that the project received the needed feedback already in the Under Development phase and is ready to be put into production
+
+### Technical Preview
+
+#### Purpose
+
+To gather feedback on the project from potential future users and extenders. The further minor releases to the projects in technical preview may contain breaking changes. 
+
+#### Timeline
+
+As long as necessary to move the project to at least the Generally Available stage. The project shouldn't stay in this phase longer than a year. If this is the case, it's possible to return the project to Under Development.  
+
+#### Support
+
+The project is ready for the test and validation environments. The support is still provided on the best effort basis for the last version in which the technical preview was available. The project still can be moved back into Under Development or directly Archived.
+
+#### Recommended Use
+
+Put the project in testing and validation environments and provide feedback to Zowe to improve the project. 
+
+#### Possible transitions
+
+- From Technical Preview to Generally Available means that the project received the needed feedback and is ready to be put into production
+- From Technical Preview to Long Term Support means that the project is stable and mature. It belongs to the production
+- From Technical Preview to Archived means that after the feedback from the customers, the project found that it's not a good fit and the development won't continue
+
+### Generally Available
+
+#### Purpose
+
+To gather more feedback and improve the project before stabilizing it for Long Term Support. The project is stable, and we don't expect major changes, but unlike for Long Term Support, the changes may be needed. We believe that it's ok for GA products to become part of the production environments. The project may need to release major versions with breaking changes more often than the projects in Long Term Support. There should be no breaking changes in the minor releases.
```

Comment:
Would breaking changes include breaking changes in the source which would **not** affect runtime, e.g. 3rd party plugin would still work.

Reply:
I would say that this level of detail, doesn't really belong here. There is a proposal for policy that explains in detail what breaking change mean here: https://github.com/zowe/community/pull/1499

And to some extent the precise definition will be up to the squads. So I don't really have answer for the question. 

I would expect that this isn't breaking change.

Reply:
Thanks for the clarification.